SUMMARY.md

```diff
@@ -1,28 +1,34 @@
 # Table of contents
 
 * [Introduction](README.md)
-* [Edit this Documentation](edit-this-documentation.md)
-* [Installation](installation.md)
+* [Installation](installation/README.md)
+  * [Docker Installation](installation/installation-docker.md)
+  * [Local Installation](installation/installation-local.md)
 * [Backend](backend/README.md)
```

Comment:
When I visit https://docs.human-connection.org/human-connection/~/revisions/-LdEQ_-L4kwVw-mn1ibR/v/docs-improve_installation_instructions/backend which is the backend overview page, I see a blank page with links to the subpages.

![Screenshot (86)](https://user-images.githubusercontent.com/2110676/56662659-ce1a2380-66a4-11e9-96dc-6134b7197388.png)